src/parser/jobs/blm/modules/Procs.js

```diff
@@ -80,19 +74,18 @@ export default class Procs extends Module {
 	}
 
 	_onGainProc(event) {
-		const status = getStatus(event.ability.guid)
+		const statusId = getStatus(event.ability.guid).id
```

Comment:
If you literally only use the ID, then `event.ability.guid` is already that ID. You don't need to check if it exists in the next block either that way. If you do fetch the object, it's good to validate it before calling on members of it just in case the result of `getStatus()` is bad, so more like this:

```
const status = getStatus(event.ability.guid)

if (!status) {
  return
}

const tracker = this._buffWindows[status.id]
```

Or alternatively:
```
const statusId = event.ability.guid
const tracker = this._buffWindows[statusId]
```